frontend/src/pages/AboutPage.tsx

```diff
@@ -1,6 +1,28 @@
-import { Text } from "@chakra-ui/layout";
+import { Container, Heading, Text } from "@chakra-ui/layout";
 import React from "react";
 
+import MainPanel from "../components/MainPanel";
+import { headingFontSizeBreakpoints } from "../styles/customTheme";
+
 export default function AboutPage() {
-  return <Text fontSize="5xl">About Us</Text>;
+  return (
```

Comment:
Our about us content has updated now, this is on the home page now and it is mostly about the team and joining it, I would say we can leave it but the main issue is we are not on this business model anymore, We can maybe replace this with the our story in the google doc, it does not even have to look super nice yet since we are focusing on the home page, but we should at least not have incorrect content